cluster/manifests/skipper/daemonset.yaml

```diff
@@ -59,6 +59,7 @@ spec:
           - "-lb-healthcheck-interval=3s"
           - "-metrics-flavour=codahale,prometheus"
           - "-enable-connection-metrics"
+          - "--max-audit-body=0"
```

Comment:
should be a single dash ("-") instead of double ("--")

Reply:
thanks fixed